src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -163,9 +163,7 @@ private HostAndPort initSentinels(Set<String> sentinels, final String masterName
         log.warning("Cannot get master address from sentinel running @ " + hap + ". Reason: " + e
             + ". Trying next one.");
       } finally {
-        if (jedis != null) {
```

Comment:
I will check again but I thought jedis is never null here.

On Fri, Jun 3, 2016 at 10:17 PM, Marcos Nils <notifications@github.com>
wrote:

> In src/main/java/redis/clients/jedis/JedisSentinelPool.java
> <https://github.com/xetorthio/jedis/pull/1303#discussion_r65761469>:
>
> > @@ -163,9 +163,7 @@ private HostAndPort initSentinels(Set<String> sentinels, final String masterName
> >          log.warning("Cannot get master address from sentinel running @ " + hap + ". Reason: " + e
> >              + ". Trying next one.");
> >        } finally {
> > -        if (jedis != null) {
>
> Same as https://github.com/xetorthio/jedis/pull/1303/files#r65761426
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xetorthio/jedis/pull/1303/files/ae11c9480520fccbb006890323b55076a526dace#r65761469>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BOfL1AB3id3-WBdRc9F-w5D5m8Nmks5qIH2ygaJpZM4ItSRz>
> .
>